include/loop_tool/backend.h

```diff
@@ -91,4 +91,69 @@ struct RegisterBackend {
 
 void loadLibrary(std::string lib_name);
 
+#ifndef DABUN_ISA
```

Comment:
would it be it be possible to keep this in the `loop_nest.cpp` or even export from the `loop_nest` library itself? It looks copy pasted a little bit

Reply:
I was unsure how to include things from loop_nest, so I've copy-pasted it to make sure it works first. Looking forward to discuss this on Zoom.

Reply:
lets get rid of this file :)

Reply:
This file was here before me, I just added DABUN_ISA... I will delete just what I added.